metrics/core.go

```diff
@@ -117,3 +94,56 @@ func CreateMetrics() *Metrics {
 
 	return m
 }
+
+func CreatePrometheusMetrics(cfg config.PrometheusMetrics) *PrometheusMetrics {
+	cacheWriteTimeBuckts := []float64{0.001, 0.002, 0.005, 0.01, 0.025, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5, 1}
+	registry := prometheus.NewRegistry()
+	promMetrics := &PrometheusMetrics{
+		Registry: registry,
+		RequestDurationMetrics: newHistogramVector(cfg, registry,
+			"Request duration",
+			"Duration in seconds to write to Prebid Cache labeled by get or put method and current URL or backend request type.",
+			[]string{"method", "result"},
+			cacheWriteTimeBuckts,
+		),
+		MethodToEndpointMetrics: newCounterVecWithLabels(cfg, registry,
```

Comment:
Similarly, consider breaking out the `method` dimension and using this to count the number of requests and labeling them with a status such as bad and error requests.

Reply:
corrected